README.md

```diff
@@ -42,42 +42,45 @@ The distinction is observable in the following scenario, with files `parent.html
 
 ```html
 <script>
-  window.foo = 'a';
-  window.getGlobalThis = () => globalThis;
+  globalThis.foo = 'a';
+  globalThis.getGlobalThis = () => globalThis;
+</script>
 </script>
 ```
 
 `frame-b.html` has the following source code:
 
 ```html
 <script>
-  window.getGlobalThis = () => globalThis;
+  globalThis.getGlobalThis = () => globalThis;
 </script>
 ```
 
-`parent.html`'s source code is:
+`parent.html`’s source code is:
 
 ```html
 <iframe src="frame-a.html"></iframe>
 <script>
-  const frame = frames[0];
-  // The global variable `foo` exists.
-  console.assert(frame.foo === 'a');
-  const before = frame.getGlobalThis();
-  frame.src = 'b.html';
-  frame.onload = () => {
-    // The global variable `foo` has disappeared.
-    console.assert(frame.foo === undefined);
-    const after = frame.getGlobalThis();
-    // But, `globalThis` still has the same identity.
-    console.assert(before === after);
+  const iframe = document.querySelector('iframe');
+  iframe.onload = () => {
+    // The global variable `foo` exists.
+    console.assert(frames[0].foo === 'a');
+    const before = frames[0].getGlobalThis();
+    iframe.onload = () => {
+      // The global variable `foo` has disappeared.
+      console.assert(frames[0].foo === undefined, 'The global object changes during navigation');
+      const after = frames[0].getGlobalThis();
+      // But, `globalThis` still has the same identity.
+      console.assert(before === after, 'globalThis maintains its identity during navigation');
+    };
+    iframe.src = 'frame-b.html';
   };
 </script>
 ```
 
-This shows that the global variable `foo` was being stored on the actual global object, which has changed during navigation, but `globalThis` has not changed during navigation. Therefore, `globalThis` is not the global object.
+[This demo](https://bead-pancake.glitch.me/) shows that the global variable `foo` was being stored on the actual global object, which has changed during navigation, but `globalThis` has not changed during navigation. Therefore, `globalThis` is not the global object.
 
-Thus, `globalThis` is observably different from “the global object”, which is not directly accessible from JavaScript.
+Thus, `globalThis` is observably different from “the global object”, which is not directly accessible from JavaScript. [In web browsers, it’s even possible that (even in the global scope), `foo !== globalThis.foo`.](https://concise-walker.glitch.me/)
```

Comment:
I suppose this is also possible with a getter, but hey. Maybe "(even in the global scope and in the absence of getters)"?